core/controllers/suggestion.py

```diff
@@ -148,6 +148,15 @@ def post(self):
 
         self.render_json(self.values)
 
+    @acl_decorators.can_suggest_changes
+    def put(self, suggestion_id):
+        try:
+            suggestion_services.update_suggestion(
+                suggestion_id, self.payload.get('translation_html'))
+        except utils.ValidationError as e:
+            raise self.InvalidInputException(e)
+        self.render_json(self.values)
+
```

Comment:
Note: We already have a handler ResubmitSuggestionHandler to edit suggestions, not sure whether we can use that?

Reply:
`ResubmitSuggestionHandler ` requires some of the other params rather just to update the suggestion. Hence I preferred to keep it on the `SuggestionHandler`

Reply:
ResubmitSuggestionHandler is slightly different as it is resubmitting an already rejected suggestion specifically.

For a more general update, I think adding a put method to the SuggestionHandler makes sense from a CRUD perspective. In a follow-up, we'll want to generalize this past just translations. Hasitha, can you add a TODO comment linking to the issue nothing this?

Reply:
Done